sdk/communication/azure-communication-phonenumbers/test/test_sip_routing_client_e2e.py

```diff
@@ -7,14 +7,18 @@
 from _shared.testcase import CommunicationTestCase
 from _shared.helper import URIReplacerProcessor
 from _shared.utils import create_token_credential, get_http_logging_policy
+from sip_routing_helper import DomainReplacerProcessor, get_user_domain, trunks_are_equal, routes_are_equal
 
 from azure.communication.phonenumbers.siprouting import SipRoutingClient, SipTrunk, SipTrunkRoute
 from azure.communication.phonenumbers._shared.utils import parse_connection_str
 
 class TestSipRoutingClientE2E(CommunicationTestCase):
-    TRUNKS = [SipTrunk(fqdn="sbs1.sipconfigtest.com", sip_signaling_port=1122), SipTrunk(fqdn="sbs2.sipconfigtest.com", sip_signaling_port=1123)]
-    ROUTES = [SipTrunkRoute(name="First rule", description="Handle numbers starting with '+123'", number_pattern="\\+123[0-9]+", trunks=["sbs1.sipconfigtest.com"])]
+    user_domain = get_user_domain()
+    additionalTrunkFqdn = "sbs3." + user_domain
 
+    TRUNKS = [SipTrunk(fqdn="sbs1." + user_domain, sip_signaling_port=1122), SipTrunk(fqdn="sbs2." + user_domain, sip_signaling_port=1123)]
```

Comment:
Relying on the list's order can be harmful since it's easy to change it and index-based value access is wide-spread in this test class.

I think that having something like this is cleaner and safer:

```
primary_trunk = f'sbs1.{user_domain}'
secondary_trunk = f'sbs2.{user_domain}'
tertiary_trunk = f'sbs3.{user_domain}'
```

Reply:
Changed to specific variables.